githttp.go

```diff
@@ -216,7 +226,16 @@ func sendFile(content_type string, hr HandlerReq) error {
 	return nil
 }
 
-func (g *GitHttp) getGitDir(file_path string) (string, error) {
+func (g *GitHttp) getGitDir(file_path string, req *http.Request) (string, error) {
+
+	var err error
```

Comment:
You can move `var err error` inside the `if` statement, it's not needed outside.